public/pages/workflow_detail/workspace/resizable_workspace.tsx

```diff
@@ -21,48 +33,149 @@ const COMPONENT_DETAILS_PANEL_ID = 'component_details_panel_id';
  * panels - the ReactFlow workspace panel and the selected component details panel.
  */
 export function ResizableWorkspace(props: ResizableWorkspaceProps) {
+  // Component details side panel state
   const [isOpen, setIsOpen] = useState<boolean>(true);
```

Comment:
nit: could just change to more descriptive name here like `isDetailsPanelOpen`

Reply:
Agreed - updated.